modules/gitops-promoting-the-rollout.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * aargo_rollouts/getting-started-with-argo-rollouts.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-promoting-the-rollout_{context}"]
+= Promoting the rollout
+
+Because your rollout is now in a paused status, as a cluster administrator, you must now manually promote the rollout to allow it to progress to the next step. 
+
+.Procedure 
+
+. Simulate another new canary version of the application by running the following command in the Argo Rollouts CLI:
++
+[source,terminal]
+----
+$ oc argo rollouts promote rollouts-demo -n <namespace> <1>
+----
+<1> Specify the namespace where the `Rollout` resource is defined.
++
+.Sample output
+[source,terminal]
+----
+rollout 'rollouts-demo' promoted
+----
+This increases the traffic weight to 40% in the canary version.
+
+. Verify that the rollout progresses through the rest of the steps, by running the following command: 
++
+[source,terminal]
+----
+$ oc argo rollouts get rollout rollouts-demo -n <namespace> --watch <1>
+----
+<1> Specify the namespace where the `Rollout` resource is defined.
++
+Because the rest of the steps as defined in the `Rollout` CR have set durations, for example, `pause: {duration: 45}`, the Argo Rollouts controller will wait that duration and then automatically move to the next step.
```

Comment:
```suggestion
Because the rest of the steps as defined in the `Rollout` CR have set durations, for example, `pause: {duration: 45}`, the Argo Rollouts controller waits that duration and then automatically move to the next step.
```
no future tense, all present